torch/_torch_docs.py

```diff
@@ -3508,25 +3508,164 @@ def merge_dicts(*dicts):
            r"""
 gradient(input, *, spacing=None, dim=None, edge_order=1) -> List of Tensors
```

Comment:
Should we change this to be `spacing=1` to clarify the default behavior?

Reply:
Thanks for the suggestion, i actually agree that it's an nice idea :)